ui/00_message/ui/job_skill_manager.xml

```diff
@@ -371,17 +371,21 @@ to level it up?</translation>
   <message>
     <key>set_enable_condition</key>
     <original>セット可能条件</original>
+    <translation>Set Conditions</translation>
   </message>
   <message>
     <key>skill_default</key>
     <original>デフォルト</original>
+    <translation>Default</translation>
   </message>
   <message>
     <key>error_not_reach_condition</key>
     <original>使用条件を満たしていません</original>
+    <translation>You do not meet the conditions to use it</translation>
   </message>
   <message>
     <key>guide_change_ex_custom_skill</key>
     <original>EXスキル切り替え</original>
+    <translation>Switching Ex Skills</translation>
```

Comment:
This is probably "Switch to Ex Skills", would have to check in the UI in-game to be sure.